### ISSUE TITLE
how to change port pigallery2 

### ISSUE THREAD

**Issue by arr111** [bpatrik/pigallery2#821]
can I change port 80 on pigallery2?
![Screenshot_2024-01-24-15-52-42-269_com opera browser](https://github.com/bpatrik/pigallery2/assets/126553612/d724efea-5237-48ef-baec-8bc198ed1c3e)

my stack on below

version: '3'
services:
  pigallery2:
    image: bpatrik/pigallery2:latest
    container_name: pigallery2
    environment:
      - NODE_ENV=production # set to 'debug' for full debug logging
      # - NODE_OPTIONS=--enable-source-maps # enable source map support on the backend for development
    volumes:
      - /opt/pigallery2/config:/app/data/config # CHANGE ME
      - "db-data:/app/data/db"
      - /opt/pigallery2/images:/app/data/images:ro # CHANGE ME, ':ro' means read-only
      - /mnt/sda1/test:/app/data/images:ro
      - /opt/pigallery2/tmp:/app/data/tmp # CHANGE ME
    ports:
      - 2280:2280
    restart: always

volumes:
  db-data:

I have changed the port to 2280. Why does it not work when accessing port 2280? always returns to default 80.

**Comment by martadinata666** [comment]
Docker-wise, application internal port doesn't need really to be bothered with, the one you want is `2280:80`. You can change to `2280:2280` but you need to change pigallery port settings to `2280`, `the pigallery configuration itself to bind to 2280`

**Comment by arr111** [comment]
> Docker-wise, application internal port doesn't need really to be bothered with, the one you want is `2280:80`. You can change to `2280:2280` but you need to change pigallery port settings to `2280`, `the pigallery configuration itself to bind to 2280`

thanks sir +62
thats worked
finally i can change the port to 2280

**Comment by arr111** [comment]
> Docker-wise, application internal port doesn't need really to be bothered with, the one you want is `2280:80`. You can change to `2280:2280` but you need to change pigallery port settings to `2280`, `the pigallery configuration itself to bind to 2280`

![Screenshot_2024-01-24-19-12-59-187_com opera browser](https://github.com/bpatrik/pigallery2/assets/126553612/e3d2dc3f-2b38-4f5d-bb62-388752ec6aa3)
this is normal?? why my container pigallery2 status changed from healty to unhealty 
![Screenshot_2024-01-24-19-14-11-750_com opera browser](https://github.com/bpatrik/pigallery2/assets/126553612/cc8661ea-1b41-4596-ba08-60ebeb1d7975)

**Comment by martadinata666** [comment]
Yes it expected as health check probing on port 80 rather than 2280.

**Comment by bpatrik** [comment]
closing this due to long inactivity, feel free to reopen.
